### DISCUSSION TITLE
Install PyPi packages on RaspPi Pico from VSCode?

### DISCUSSION THREAD

**Discussion by MarkuBu** [micropython/micropython#9334] [Using MicroPython] [answered]
Hi, I want to use VSCode for MicroPython development on the RaspPi Pico. It works so far, but I haven't figured out how to install PyPi packages like I can do in Thonny.

Is this possible? And if yes, how?

**Comment by jimmo** [comment, marked as answer]
I don't know about VSCode specifically, but package installation is something that is currently being worked on.

If you want to replicate what Thonny does, the author of Thonny also provides the `pipkin` tool which does just the package installation part. https://github.com/aivarannamaa/pipkin

**Reply by MarkuBu** [reply]
Sounds good. Will try it asap. Thanks

**Comment by TA1DB** [comment]
Hi @MarkuBu 

I would like to do the same; building debugging upolading Micropython for RPi Pico on Vscode. Could you please explaing shortly how do you do it? Which extensin(s) you're using? Do you make debugging, uploading? How? Are you satisfied?
